<?xml version="1.0"?>
<launch>
  <arg name="robot_name" default=""/>
    <!-- You can change control program  -->

    <include file="$(find burger_navigation)/launch/your_burger_navigation_sample.launch">
        <arg name="robot_name" value="$(arg robot_name)" />
    </include>

    <node pkg="burger_war" type="myRobot.py" name="MyRobot" output="screen"> 
        <param name="robot_name" value="$(arg robot_name)" />
    </node>

    <!-- End of your space  -->
</launch>

